Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV ELSTER_DATENLIEFERANT=$elster_datenlieferant
 ENV ELSTER_HERSTELLER_ID=$elster_hersteller_id
 
 WORKDIR /app
-RUN echo "APT::Default-Release \"bullseye\";" >> /etc/apt/apt.conf.d/default-release && \
+RUN echo "Package: *\nPin: release n=bookworm\nPin-Priority: 50\n" >> /etc/apt/preferences && \
```

Comment:
I would have assumed that we would raise the pin priority of bullseye but we are setting the priority of bookworm quite low, correct? 


LGTM

Reply:
This is the solution proposed in the SO thread you mentioned. We're assigning a lower priority to bookworm so that apt would never prefer it to the bullseye sources unless explicitly instructed to.